README.md

```diff
@@ -26,6 +26,28 @@ cd GenAIComps
 pip install -e .
 ```
 
+- Install Data Science Stack on your workstation
```

Comment:
Why we need this in GenAIComps main page?

Reply:
@andreeamun my apologies for this late response, I thought I had commented on this PR earlier. OPEA GenAIExamples are sample workflows.  The Data Science Kit and MLOps solution such as Charmed KubeFlow or upstream Kubeflow are excellent additions but belong really in GenAIExamples with an illustration of how they can be used to solve a real problem. The edits you made in the README.md would move into that your GenAIExample README. This would be a first sample that is more developer experimentation specific. Given it will be rather large, our policy has been to propose the feature via a PR in our docs sub-project. RFC: XYZ.  https://github.com/opea-project/docs/pull/222. We will watch for it and jump right on to review. We appreciate your interest and contribution to OPEA.